### ISSUE TITLE
pretty print XML ElementTree

### ISSUE THREAD

**Issue by samuelcolvin** [samuelcolvin/python-devtools#79]
Display it as XML, something like [this](https://stackoverflow.com/a/28814053/949890).

**Comment by sshishov** [comment]
What is this issue about? Could you please elaborate? What should be pretty printed? the XML file, the string which can be parsed by XML or the full output should be converted to XML?

**Comment by alexmojaki** [comment]
I think objects of type `xml.etree.ElementTree`

**Comment by samuelcolvin** [comment]
exactly, currently this library doesn't know how to pretty print them I think.
